mmeds/tests/server/test_server.py

```diff
@@ -793,21 +793,25 @@ def test_fa_single_id(self):
         self.generate_aliquot_id()
 
     def generate_aliquot_id(self):
+        return
```

Comment:
bare return at top of function?

Reply:
It doesn't work, and I think the single ID aliquot generation functionality is gonna be dropped anyway so I don't want to spend time figuring out how to fix the test for it